indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/models/IndexedStorageManifest.scala

```diff
@@ -38,31 +50,34 @@ case class IndexedStorageManifest(
   version: Int,
   createdDate: Instant,
   payloadFiles: Seq[IndexedFileFields],
-  payloadFileCount: Int,
-  payloadFileSize: Long,
-  payloadFileSuffixTally: Seq[IndexedSuffixTally],
-  newPayloadFileCount: Int,
-  newPayloadFileSize: Long,
-  newPayloadFileSuffixTally: Seq[IndexedSuffixTally],
+  payloadStats: IndexedPayloadStats,
   @JsonKey("type") ontologyType: String = "Bag"
 )
 
 object IndexedStorageManifest {
   def apply(storageManifest: StorageManifest): IndexedStorageManifest = {
-    val payloadFiles = storageManifest.manifest.files.map(IndexedFileFields(_))
+    val storageManifestFiles = storageManifest.manifest.files
+    val payloadFiles = storageManifestFiles.map(IndexedFileFields(_))
+
+    val fileSuffixTally = FileSuffixCounter.count(storageManifestFiles)
+    val indexedSuffixTally = fileSuffixTally
+      .map(IndexedSuffixTally(_))
+      .toList
+      .sortBy(_.suffix)
```

Comment:
I'd sort by count instead, so when I'm looking in the reporting cluster the most common file types float to the top.

Reply:
reverse count then I think - so highest first.